src/data/polynomial/derivative.lean

```diff
@@ -177,6 +177,15 @@ begin
   exact ih _ this (this.trans_le $ nat.le_of_lt_succ hx) rfl
 end
 
```

Comment:
Unless there's good reason to remove them, mathlib PRs shouldn't lose results. You should make sure all the lemmas from the file you deleted are proved in some form, unless inlining `iterated_deriv` makes them completely trivial.

Reply:
That was certainly the intention. Did I miss any?

Reply:
Oh, I was just surprised at the size of the diff. I just realized you mentioned some lemmas already existed.